README.rst

```diff
@@ -1,85 +1,61 @@
+.. raw:: html
+
+   <p align="center">
+     <img
+       src="https://img.shields.io/github/workflow/status/omnivector-solutions/jobbergate/test_on_push/main?label=main-build&logo=github&style=plastic"
+       alt="main build"
+     />
+     <img
+       src="https://img.shields.io/github/issues/omnivector-solutions/jobbergate?label=issues&logo=github&style=plastic"
+       alt="github issues"
+     />
+     <img
+       src="https://img.shields.io/github/issues-pr/omnivector-solutions/jobbergate?label=pull-requests&logo=github&style=plastic"
+       alt="pull requests"
+     />
+     <img
+       src="https://img.shields.io/github/contributors/omnivector-solutions/jobbergate?logo=github&style=plastic"
+       alt="github contributors"
+     />
+   </p>
+   <p align="center">
+     <img
+       src="https://img.shields.io/pypi/pyversions/jobbergate?label=python-versions&logo=python&style=plastic"
+       alt="python versions"
+     />
+     <img
+       src="https://img.shields.io/pypi/v/jobbergate?label=pypi-version&logo=python&style=plastic"
+       alt="pypi version"
+     />
+     <img
+       src="https://img.shields.io/pypi/l/jobbergate?style=plastic"
```

Comment:
Jobbergate itself is not at pypi, we need some sub-project here to make it work.

```suggestion
       src="https://img.shields.io/pypi/l/jobbergate-cli?style=plastic"
```

Reply:
Removed the whole block in commit d2ca89e.